week2/1-tuesday/class/exercise10/exercise10.js

```diff
@@ -0,0 +1,9 @@
+function fiveOrThree(number){
+    if(number % 5 === 0 || number % 3 === 0){
+        return true;
+    } else {
+        return false;
+    }
+}
```

Comment:
This works but there are two edge cases it won't account for:

- something other than a number is passed in
- a zero is passed in

Update and test with this:

```suggestion
    if(typeof number !== "number") return 'Sorry, you must use a number. Please try again.'
    if(!number) return 'Sorry, we do not accept zeros';
    if(number % 5 === 0 || number % 3 === 0){
        return true;
    } else {
        return false;
    }
}
```

The `!number` may be confusing but it's testing to see if `number` is false and `0` is considered `false`.